presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -321,7 +321,7 @@ public PlanNode visitJoin(JoinNode node, RewriteContext<Expression> context)
                     leftPredicate = leftOuterJoinPushDownResult.getOuterJoinPredicate();
```

Comment:
Can you explain the purpose of this commit? Maybe add the explanation to the commit message. I can see what it's doing, but I can't tell what it's for or why it's needed.